src/main.cpp

```diff
@@ -2732,6 +2738,8 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
     // Grab the consensus branch ID for the block's height
     auto consensusBranchId = CurrentEpochBranchId(pindex->nHeight, chainparams.GetConsensus());
 
+    size_t total_shielded_tx = 0;
```

Comment:
this amount is still debated in the original zip220 proposal @str4d

Reply:
ZIP 221 now states that this field in the node will count only Sapling shielded transactions.

Reply:
So this is correct now below:

```rust
        if (tx.vShieldedOutput.size() > 0) {
            total_shielded_tx += 1;
        }
```